cmd/wallet-web/src/pages/CHAPISharePage.vue

```diff
@@ -67,11 +66,15 @@ onMounted(async () => {
     return;
   }
   const credentials = toRaw(presentation.value.verifiableCredential);
-  const filteredCreds = filterCredentialsByType(credentials, [manifestCredType]);
-  filteredCreds.map((credential) => {
-    const manifest = getManifest(credential);
-    const processedCredential = getCredentialDisplayData(credential, manifest);
-    credsFound.value.push({ ...processedCredential, showDetails: false });
+  let manifest = prepareCredentialManifest(
+    presentation.value,
+    credentialManifests.value,
+    requestOrigin
+  );
+  manifest.issuer.id = manifest.issuer.id.value;
```

Comment:
So without this line the logic does not work?

Reply:
No, `manifest.issuer.id` is a ref for some reason
![image](https://user-images.githubusercontent.com/44453261/172663177-f595c3d2-e39b-43cc-87e1-6c1d40213289.png)